ras_party/controllers/enrolments_controller.py

```diff
@@ -20,11 +22,50 @@ def respondent_enrolments(
     session: session, party_uuid: UUID, business_id: UUID = None, survey_id: UUID = None, status: int = None
 ) -> list[Enrolment]:
     """
-    returns a list of respondent Enrolments. Business_id, survey_id and status can also be added as conditions
+    returns a list of respondent enrolments with business and survey details
     """
 
     respondent = query_respondent_by_party_uuid(party_uuid, session)
     if not respondent:
         raise NoResultFound
 
-    return query_respondent_enrolments(session, respondent.id, business_id, survey_id, status)
+    enrolments = query_respondent_enrolments(session, respondent.id, business_id, survey_id, status)
+
+    if not enrolments:
+        return []
+
+    surveys_details = get_surveys_details()
+    respondents_enrolled = []
+    for enrolment, business_ref, business_attributes in enrolments:
+        survey_id = enrolment.survey_id
+        respondents_enrolled.append(
+            (
+                {
+                    "enrolment_status": enrolment.status.name,
+                    "business_details": {
+                        "id": enrolment.business_id,
+                        "name": business_attributes["name"],
+                        "trading_as": business_attributes["trading_as"],
+                        "ref": business_ref,
+                    },
+                    "survey_details": {
+                        "id": survey_id,
+                        "long_name": surveys_details.get(survey_id)["long_name"],
+                        "short_name": surveys_details.get(survey_id)["short_name"],
+                        "ref": surveys_details.get(survey_id)["ref"],
+                    },
+                }
+            )
+        )
+    return respondents_enrolled
+
+
+@with_query_only_db_session
+def is_respondent_enrolled(party_uuid: UUID, business_id: UUID, survey_id: UUID, session: session) -> bool:
```

Comment:
I've moved this, with the aim to shift anything associated to enrolments to the enrolments controller